README.md

```diff
@@ -16,11 +16,9 @@ Add the plugin to your Webpack configuration:
 
   resolve: {
     plugins: [new BowerResolvePlugin()],
-    modules: [
-        'bower_components'
-    ],
-    descriptionFiles: ['bower.json', 'package.json'],
-    mainFields: ['main', 'browser']
+    modules: ['bower_components', 'node_modules'],
+    descriptionFiles: ['.bower.json', 'package.json'],
```

Comment:
We can merge, just I would prefer to keep bower.json without a dot for a while if you don't mind, as it works in most cases now.

 ``enhanced-resolve`` is being exclusively used now as a resolve engine for webpack 2.x.

p.s. a bit unrelated but [see the issue](https://github.com/webpack/enhanced-resolve/issues/80) . If you could figure out the way on how or it's possible to resolve multiple files per one webpack require, it would be nice to have aswell. It's more of the bower thing, as it tends to include css files in the main attribute aswell, so we wouldn't need to load them separately (e.g. select2, bootstrap, etc..) .

Please share your thoughts or ideas if you are able to find anything. :-)

Thanks!